src/data.js

```diff
@@ -138,6 +138,23 @@ module.exports = [
       'AWS',
       'cPanel WHM',
     ],
+  },
+   {
+    name: 'Tugrul Erdem Dogru',
+    description: 'Jr Front-End & Historian ',
+    url: 'http://tugrulerdemdogru.com/uses',
+    twitter: '@tugrulerdemd',
+    emoji: '🎉',
+    country: '🇹🇷',
+    computer: 'HP Pavilion',
```

Comment:
Error: "computer" must be one of [apple, windows, linux]

Reply:
changed